apps/extension/src/ui/domains/Portfolio/AccountSelect.tsx

```diff
@@ -295,7 +295,7 @@ const Item = forwardRef<HTMLDivElement, ItemProps>(function Item(
       {t("All Accounts")}
     </div>
   ) : (
-    <div className="flex w-full items-center gap-2">
+    <div className="flex items-center gap-2">
       <div className="overflow-hidden text-ellipsis whitespace-nowrap text-sm">{item.name}</div>
```

Comment:
removing `w-full` would remove the truncate feature and make the account icon disappear.
instead need to keep it and just use `justify-center` to ensure content is centered
```suggestion
    <div className="flex w-full items-center justify-center gap-2">
```

Reply:
sorry please ignore this suggestion, vscode bug i guess

Reply:
this is what i meant : `flex w-full items-center justify-center gap-2`

Reply:
ah awesome! maybe `md:justify-center lg:justify-start` though?

Reply:
oops yes, good catch !